website/content/commands/config/delete.mdx

```diff
@@ -7,10 +7,20 @@ page_title: 'Commands: Config Delete'
 
 Command: `consul config delete`
 
+Corresponding HTTP API Endpoint: [\[DELETE\] /v1/config/:kind/:name](https://www.consul.io/api-docs/config#delete-configuration)
+
 The `config delete` command deletes the configuration entry specified by the
 kind and name. See the [configuration entries docs](/docs/agent/config-entries)
 for more details about configuration entries.
 
+The table below shows this command's [required ACLs](/api#authentication). Configuration of
+[blocking queries](/api/features/blocking) and [agent caching](/api/features/caching)
+are not supported from commands, but may be from the corresponding HTTP endpoint.
+
+| ACL Required                        |
+| ----------------------------------- |
+| `service:write` or `operator:write` |
```

Comment:
Should this be `or` or `and`? `or` seems like it would make sense, but I've never looked into what's actually implemented. The corresponding HTTP API endpoint docs use `,` to connect them ... which is ambiguous (could or `or` or `and`). Thanks for making this unambiguous!

Reply:
Hmm... this and the other configuration endpoints are strange. Going to the HTTP API endpoint docs, it seems that the ACL required actually depends on the config entry being edited. `service:read` seems to be what's needed for most config entries, but not all. This is handled with a "footnote" in the HTTP API endpoint docs. Should we do that here as well?

![image](https://user-images.githubusercontent.com/85913323/148857009-176b5e14-d6e5-4365-b5e1-e26fdcf3d9f1.png)